mne/evoked.py

```diff
@@ -291,7 +291,8 @@ def save(self, fname):
         write_evokeds(fname, self)
 
     def __repr__(self):
-        s = "comment : '%s'" % self.comment
+        s = ('kind :  %s, ' % self.kind) if self.kind != 'average' else ''
```

Comment:
I would find it more systematic to be symmetric here, imagine you have a study with different kinds and you print them,  it's nicer to have things equal.